purchase_request_to_requisition/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License AGPL-3
+
+Purchase Request to Call for Bids
+=================================
+This module introduces the following features:
+* The possibility to create new Call for Bids or update existing Bids from
+Purchase Request Lines.
+* The possibility to create Purchase Requests on the basis of Procurement
+Orders.
+
+Installation
+============
+
+No specific instructions required.
+
+
+Configuration
+=============
+
+No specific instructions requied.
+
+Usage
+=====
+Create Purchase Requests from Procurement Orders
+------------------------------------------------
+Go to the product form, in tab 'Procurement' and choose 'Purchase Request'.
+When a procurement order is created and the supply method is 'Make to Order'
+the application will now create a Purchase Request.
```

Comment:
Question: why is this capability to generate purchase requests from purchase order in this purchase_request_to_requisition module? This capability is useful even if we don't use purchase_requisition. Would it not be better in a stand alone module?